.gitignore

```diff
@@ -35,3 +35,6 @@ yarn-error.log*
 
 # editor
 .vscode/
+/test-results/
```

Comment:
All playwright related. Love how playwright saves videos in `test-results`.

Reply:
Nice! Have not used before... I'm interested in learning though!